### DISCUSSION TITLE
Security feature

### DISCUSSION THREAD

**Discussion by Sovince37** [rustdesk/rustdesk#7448] [Ideas]
Hello, 
I've installed RustDesk, which is indeed a great program, but it lacks a functionality.
RustDesk is installed as a service, everything works perfectly well, especially for the UAC.
However, currently, the service installation means that RustDesk is always active and listening on its ports.
I would like it to be necessary for the user to launch RustDesk using the shortcut on their desktop (or start menu, or anything else), and for the main RustDesk window to be open.
I've already tested disabling the "Rustdesk service"; it works, but I lose access to the UAC.

Is it possible to add an option like this : "display the authorization popup only when the main window is open"?

**Comment by rustdesk** [comment]
This is a good example if you want a feature request get high priority.
https://github.com/rustdesk/rustdesk/discussions/7437

**Comment by rustdesk** [comment]
close as dupllcate: https://github.com/rustdesk/rustdesk/discussions/7033
